release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -318,6 +318,12 @@ With this release, some Operators managed by Operator Lifecycle Manager (OLM) on
 For more information, see _Token authentication for Operators on cloud providers_.
 //xref :../operators/operator_sdk/osdk-token-auth.adoc#osdk-token-auth[Token authentication for Operators on cloud providers].
 
+==== Authentication Operator honors `noProxy` when configured
```

Comment:
Should this mention that it's for connection checks?

```suggestion
==== Authentication Operator honors `noProxy` during connection checks
```

Reply:
Changed, thanks!